pkg/backends/influxdb/backend_test.go

```diff
@@ -53,6 +53,14 @@ func TestBackend(t *testing.T) {
 				`sleep_analysis_hr,target_name=test asleep=6.108333333333333,inBed=6.809728874299261,inBedEnd="2021-12-18T09:04:45+08:00",inBedSource="iPhone",inBedStart="2021-12-18T02:12:50+08:00",sleepEnd="2021-12-18T08:57:06+08:00",sleepSource="Irvin’s Apple Watch",sleepStart="2021-12-18T02:21:06+08:00" 1639789416000000000`,
 			},
 		},
+		{
+			name:    "write non aggregated sleep analysis metrics",
+			payload: fixtures.PayloadMetricsSleepAnalysisNonAggregated,
+			wantMetrics: []string{
+				`sleep_analysis_hr,source=Irvin's\ Apple\ Watch,target_name=test,value=Core state=1i 1639765266000000000`,
+				`sleep_analysis_hr,source=Irvin's\ Apple\ Watch,target_name=test,value=Core qty=6.108333333333333,state=0i 1639789026000000000`,
```

Comment:
Well, it does still  track `hr`s of _sleep analysis_ in the `qty` field. The `state` field is not tracking `hr`s.

Getting into specific database differences, I'm using [https://github.com/VictoriaMetrics/VictoriaMetrics](VictoriaMetrics), a drop-in replacement for InfluxDB. VM actually only allows a single field per "metric", and translates this influx measurement into these metrics:

- `sleep_analysis_hr_state`
- `sleep_analysis_hr_qty`

Since this is the case for my database, I'm perfectly happy with both multiple measurements with single fields, and different measurements for aggregated and non-aggregated sleep analysis.

I think I'll hold off on merging and modify this PR to introduce multiple measurements and to differentiate aggregated and non-aggregated sleep analysis. I appreciate how you are seemingly anticipating my needs here before I even voice them! 😂

Reply:
> I think I'll hold off on merging and modify this PR to introduce multiple measurements and to differentiate aggregated and non-aggregated sleep analysis.

In this case, maybe we could simply introduce a different measurement name for non-aggregated sleep data in this PR, and then make any necessary changes for aggregated sleep data in #11. Don't want to hold off merging for too long! WDYT?

Reply:
Yes, that's what I'll do.

Reply:
Let me know what you think of the latest commit - I changed the debug logging to log just the metric name instead of the influx measurement name because it made it simpler. If it's good with you, this can be merged.